libzhlgen/test/functions/mac/1.6.13/Settings.zhl

```diff
@@ -0,0 +1,19 @@
+"803d8a5e1e00":
+static __amd64 bool Settings::GetDlcEnabled();
+"554889e54157415641554154534883ec684989ff488d9d78":
+static cleanup __amd64 std::string Settings::GetHotkeyName(const std::string& name);
+"554889e54157415641554154534883ec2848897db8488d47":
+static cleanup __amd64 SDLKey Settings::GetHotkey(const std::string& hotkeyName);
+"554889e54157415641554154534883ec684189f5":
+static cleanup __amd64 void Settings::SetHotkey(const std::string& hotkeyName, SDLKey key);
+"554889e54157415641554154534881ec58010000488dbdf0":
+static cleanup __amd64 void Settings::SaveSettings();
+"554889e58a05????????5d":
+forceDetour static __amd64 char Settings::GetCommandConsole();
```

Comment:
See other comments on `forceDetour` need to evaluate if it's actually needed for this function signature on Mac.